app/models/family.rb

```diff
@@ -86,6 +86,10 @@ def everyone_has_birthdates?
   def anyone_has_email?
     attendees.any? { |p| p.email.present? }
   end
+ 
+  def anyone_registered_for_legacy_conferences()
```

Comment:
Functions that return a boolean ought to have `?` in their names,
```suggestion
  def anyone_registered_for_legacy_conferences?()
```

Reply:
good catch :S